components/ILIAS/Saml/Saml.php

```diff
@@ -32,6 +32,7 @@ public function init(
         array | \ArrayAccess &$pull,
         array | \ArrayAccess &$internal,
     ): void {
-        // ...
+        $contribute[Component\Resource\PublicAsset::class] = fn() =>
```

Comment:
The `SAML` component should also provide endpoints locaed in `components/ILIAS/Saml/lib`:

* metadata.php
* saml1-acs.php
* saml2-acs.php
* saml2-logout.php